### DISCUSSION TITLE
Why was no_gaps_when_only removed?

### DISCUSSION THREAD

**Discussion by rayhan-ryu** [hyprwm/Hyprland#8527] [Q&A]
I just updated Hyprland recently and I noticed `no_gaps_when_only` is missing along with old shadow configuration. It was removed in [0.45.0](https://github.com/hyprwm/Hyprland/releases/tag/v0.45.0) and the wiki provided the equivalent [windows rules](https://wiki.hyprland.org/Configuring/Workspace-Rules/#smart-gaps) so I fixed my config in around five minutes.

My questions is that why was it removed? Is it because the window rule implementation is better or there are very few people who use it (and I'm those few people)? Thanks.

**Comment by Aqa-Ib** [comment]
I think that it was meant to simplify the code base.

**Comment by rayhan-ryu** [comment]
I don't this this question is ever that important. Closing this.
